### DISCUSSION TITLE
Can I resize an image by simply extending it with an empty background instead of upscaling it?

### DISCUSSION THREAD

**Discussion by mainrs** [chaiNNer-org/chaiNNer#2327] [Q&A]
I have an image I did some editing. I would now like to scale it by 200%. I thought the box scaler does, but I think it is just another way to upscale an image.

**Comment by joeyballentine** [comment]
You are looking for the create border/create edges nodes. A bit oddly named, but they let you expand out the image. (I might have the node names wrong since this is just from memory)
